templates/pools/pools.html

```diff
@@ -4,17 +4,26 @@
   <script src="/js/highcharts/highcharts-global-options.js"></script>
   <script src="/js/luxon.min.js"></script>
   <script>
+    let performanceChart
     if (window.location.hash == "#performance") {
       $("#distribution-chart").addClass("d-none")
       $("#performance-chart").removeClass("d-none")
+      $("#performance-chart-btn-hide-all").removeClass("d-none")
     }
     $("#distribution-chart-btn").on("click", () => {
       $("#distribution-chart").removeClass("d-none")
       $("#performance-chart").addClass("d-none")
+      $("#performance-chart-btn-hide-all").addClass("d-none")
     })
     $("#performance-chart-btn").on("click", () => {
       $("#distribution-chart").addClass("d-none")
       $("#performance-chart").removeClass("d-none")
+      $("#performance-chart-btn-hide-all").removeClass("d-none")
+    })
+    $("#performance-chart-btn-hide-all").on("click", () => {
+      performanceChart.series.forEach((series) => {
+        series.setVisible(false, false)
+      })
```

Comment:
Doesn't work for me, clicking on the button only causes this error:
![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/109136188/ab1c2f3f-2890-4f6d-bb8e-8917a75165d9)